### ISSUE TITLE
[Bug] React-translator demo gives a "No available backend found" error

### ISSUE THREAD

**Issue by sebinthomas** [xenova/transformers.js#124]
**Describe the bug**
*A clear and concise description of what the bug is.*

I cloned the repository and ran `npm install` and `npm run dev` on the react-translator demo. It loaded the models but gave a "No available backend found" error.

**How to reproduce**
*Steps or a minimal working example to reproduce the behavior*

Clone the repository, run `npm install` and `npm run dev`

**Expected behavior**
*A clear and concise description of what you expected to happen.*

Clear translation from english to french

**Logs/screenshots**
```
backend-impl.js:87 Uncaught (in promise) Error: no available backend found. ERR: [wasm] RuntimeError: Aborted(LinkError: WebAssembly.instantiate(): Import #41 module="a" function="P" error: function import requires a callable). Build with -sASSERTIONS for more info.
    at http://127.0.0.1:5173/node_modules/.vite/deps/@xenova_transformers.js?v=45fb8b31:464:21
    at async d.create (http://127.0.0.1:5173/node_modules/.vite/deps/@xenova_transformers.js?v=45fb8b31:445:110)
    at async constructSession (http://127.0.0.1:5173/node_modules/.vite/deps/@xenova_transformers.js?v=45fb8b31:19761:12)
    at async Promise.all (index 1)
    at async AutoModelForSeq2SeqLM.seq2seqLoadModel [as LOAD_FUNCTION] (http://127.0.0.1:5173/node_modules/.vite/deps/@xenova_transformers.js?v=45fb8b31:19879:14)
    at async AutoModelForSeq2SeqLM.from_pretrained (http://127.0.0.1:5173/node_modules/.vite/deps/@xenova_transformers.js?v=45fb8b31:21542:16)
    at async Promise.all (index 1)
    at async pipeline (http://127.0.0.1:5173/node_modules/.vite/deps/@xenova_transformers.js?v=45fb8b31:24137:15)
    at async http://127.0.0.1:5173/src/worker.js?type=module&worker_file:28:22
(
```

**Environment**
- Transformers.js version: ^2.0.0
- Browser (if applicable): Brave browser
- Operating system (if applicable): Mac OSX
- Other:
   *Node*: v16.20.0
   *NPM*: 8.19.4

**Additional context**
*Add any other context about the problem here.*
Potentially related to [https://github.com/microsoft/onnxruntime/issues/9322](https://github.com/microsoft/onnxruntime/issues/9322) ?

**Comment by xenova** [comment]
Hmm that's odd. Does the demo site work for you?

https://xenova.github.io/transformers.js

Admittedly, I haven't fully tested on Brave, but since it's chromium based, I'm more inclined to believe it's an issue with the build tool.

**Comment by sebinthomas** [comment]
I think its something to do with the build tool/ onnxruntime too. I tried with Node: v18.16.0 and NPM: 9.5.1 and I tried it on google chrome as well. Can you let me know which configuration works for you ?

**Comment by xenova** [comment]
Which version of onnxruntime are you using? 1.14 or 1.15?

**Comment by sebinthomas** [comment]
I'm running 1.15.0 (my package-lock.json says so) Might be that. Let me downgrade it and check

**Comment by sebinthomas** [comment]
Downgrading to 1.14.0 worked.  I was able to get it running. Guess it's an issue with version 1.15.0.

**Comment by xenova** [comment]
Alright! Thanks for investigating. I'll update it to ensure it works with 1.15.0 as well :)

**Comment by xenova** [comment]
Okay I found the issue - it's simply a mismatch between the wasm files (by default hosted on jsdelivr) and the JS library. What I'll do (so that this doesn't randomly happen in future), is freeze the onnx runtime version.

And I'll also include examples for how to host the wasm files yourself (which guarantees correctness).

**Comment by sebinthomas** [comment]
Thank you @xenova for the timely replies and the fix. Transformers.js is such an awesome package and so under-the-radar :)

**Comment by jlia0** [comment]
> Okay I found the issue - it's simply a mismatch between the wasm files (by default hosted on jsdelivr: 1.14.1) and the JS library (which you installed: 1.15.0). Using the correct wasm files fixes the issue (so that it works with 1.15.0)
> 
> What I'll do (so that this doesn't randomly happen in future), is freeze the onnx runtime version for each version, and then update the dependency version on each update.
> 
> And I'll also include examples for how to host the wasm files yourself (which guarantees correctness).

I am having the issue again, any instructions on how to host the WASM file ourselves?
